schedule/yast/yast2_ncurses/yast2_ncurses_gnome.yaml

```diff
@@ -14,9 +14,9 @@ schedule:
   - console/consoletest_setup
   - console/suseconnect_scc
   - console/setup_libyui_running_system
-  - console/yast2_i
+  # - console/yast2_i
```

Comment:
I guess these changes were not intentional, other than this LGTM

Reply:
thanks, I missed that.